test/arrays.coffee

```diff
@@ -26,6 +26,40 @@ test "incorrect indentation without commas", ->
   ok result[0][0] is 'a'
   ok result[1]['b'] is 'c'
 
+# Elisions
+test "array elisions", ->
+  eq [,1].length, 2
+  eq [,,1,2,,].length, 5
```

Comment:
No, the very last comma is the optional end comma.

Reply:
`[a,].length is 1`
`[a,,].length is 2`
`[,a].length is 2`
`[,,a].length is 3`
In this case `[,,1,2,,,]` length is 5, because two `,,` at the beginning are count as two `Elisions` and the last `,` is the optional end comma (e.g. not `Elision`).

[Example](https://babeljs.io/repl/#?babili=false&browsers=%3E%203%25&build=&builtIns=false&code_lz=IYGwpgTgLgFA2gGgQRgQJiQXQHTgHYDmUAFgJQBQQA&debug=false&circleciRepo=&evaluate=true&lineWrap=false&presets=env&prettier=false&targets=&version=6.26.0) on Babel.